nixos/modules/programs/xppen.nix

```
@@ -0,0 +1,27 @@
+{ config, lib, pkgs, ... }:
+
+with lib; let
+  cfg = config.programs.xppen;
+in
+
+{
+
+  options.programs.xppen = {
+    package = mkPackageOption pkgs "xppen" { };
+
+    enable = mkEnableOption (lib.mdDoc "XPPen PenTablet application");
```

nit: I don't think there's any hard rules here, but the `enable` option tends to come on top in most modules, as a sort of unnamed convention.

```suggestion
    enable = mkEnableOption (lib.mdDoc "XPPen PenTablet application");
    package = mkPackageOption pkgs "xppen" { };
```